src/ophyd_async/epics/pmac/_pmacIO.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import numpy.typing as npt
+from bluesky.protocols import Flyable, Preparable
+
+from ophyd_async.core import StandardReadable, SubsetEnum
+
+from ..signal.signal import epics_signal_r, epics_signal_rw
+
+
+class Pmac(StandardReadable, Flyable, Preparable):
+    """Device that moves a PMAC Motor record"""
+
+    def __init__(self, prefix: str, cs="", name="") -> None:
+        self.timeArray = epics_signal_rw(
+            npt.NDArray[np.float64], prefix + ":ProfileTimeArray"
+        )
+        self.a = epics_signal_rw(npt.NDArray[np.float64], prefix + ":A:Positions")
+        self.use_a = epics_signal_rw(bool, prefix + ":A:UseAxis")
+        self.a_vel = epics_signal_rw(npt.NDArray[np.float64], prefix + ":A:Velocities")
```

Comment:
Could we do these as `DeviceVector`s instead?
```python
        cs_letters = "ABCUVWXYZ"
        # 1 indexed CS axes so we can index into them from the compound motor input link
        self.positions = DeviceVector({
            i + 1: epics_signal_rw(npt.NDArray[np.float64], f"{prefix}{letter}:Positions")
            for i, letter in enumerate(cs_letters)
        })
        self.use_axis = ...
        self.velocities = ...
```

Reply:
Maybe we leave it as it is for now, then consider this issue along with the direct demands and raw motor port and address issue below